pages/scenario_comparison_page.py

```diff
@@ -19,3 +20,110 @@ def comparing_scenarios_page():
+        title = ["Total difference between air source heat pumps and gas boilers by installation year and scenario for average household",
+                 "(values below zero indicate air source heat pumps are cheaper)"]
+    else:
+        results["cost_difference_ashp_minus_boiler"] = (
+            results["cost_difference_ashp_minus_boiler"] / 15  # annualise over 15 years
+        )
+        title = ["Annual difference between air source heat pumps and gas boilers by installation year and scenario for average household",
+                 "(values below zero indicate air source heat pumps are cheaper)"]
+    line_chart = (
+        alt.Chart(results)
+        .mark_line(point=True, strokeWidth=3)
+        .encode(
+            x=alt.X("installation_year:O", title="Installation year"),
+            y=alt.Y("cost_difference_ashp_minus_boiler:Q", title=""),
+            color=alt.Color("scenario:N", title="Scenario", scale=alt.Scale(domain=scenarios_list, range=NESTA_COLOURS)),
+            tooltip=[
+                alt.Tooltip("installation_year:O", title="Installation year"),
+                alt.Tooltip("scenario:N", title="Scenario"),
+                alt.Tooltip("cost_difference_ashp_minus_boiler:Q", title="Cost difference (heat pump - boiler)", format=".2f"),
+            ],
+        )
+    )
+
+    # Shaded area for y < 0
+    min_value = results["cost_difference_ashp_minus_boiler"].min()
+
+    # Create dummy DataFrame covering the x-axis
+    area_df = pd.DataFrame({
+        "installation_year": results["installation_year"].unique(),
+        "y": [min_value] * len(results["installation_year"].unique())
+    })
+
+    area_below_zero = (
+        alt.Chart(area_df)
+        .mark_area(color="lightgray", opacity=0.3)
+        .encode(
+            x=alt.X("installation_year:O", title="Installation year"),
+            y=alt.Y("y:Q", title=""),
+            y2=alt.Y2(value=0)
+        )
+    )
+    label = (
+        alt.Chart(pd.DataFrame({"installation_year": [results["installation_year"].iloc[0]], "y": [-100]}))
+        .mark_text(text="Air source heat pumps are cheaper", fontSize=16, color="grey", align="left")
+        .encode(
+            x=alt.X("installation_year:O"),
+            y=alt.Y("y:Q"),
+        )
+    )
+
+    # Combine charts
+    final_chart = (area_below_zero + line_chart + label).properties(
+            width=800,
+            height=500,
+            title=title
+        ).configure_title(fontSize=20
+        ).configure_axis(labelFontSize=16, titleFontSize=16
+        ).configure_legend(labelFontSize=16, titleFontSize=16)
+    
+    st.altair_chart(final_chart, use_container_width=True)
+
+    st.markdown("### Download data")
+    csv = results.to_csv(index=False).encode('utf-8')
+    st.download_button(
+        label="Download data as CSV",
+        data=csv,
+        file_name=f'scenario_comparison_cost_decile_{cost_decile}.csv',
+        mime='text/csv',
+        icon=":material/download:",
+    )
+
```

Comment:
A really helpful piece of feedback from the team in the sprint 4 slide deck was to have a table explaining the scenarios below the chart. I think we could just copy and paste the same table that's displayed in the about data page when we've finalised the wording :)

```suggestion

    st.markdown(
        """
                ### Preset scenarios explained
                
                The assumptions underlying each preset scenario are:

                <INSERT UPDATED TABLE FROM about_data_page.py ONCE IT IS FINALISED>

                You can find more information about the methodology and data sources used on the Data and methodology page.
                """
    )
```